### PR TITLE
README.md: Add Newcomer Gamification cEP

### PR DESCRIPTION
**Pull request by sks444**
Closes https://github.com/coala/cEPs/issues/165

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,3 +19,5 @@ in the `cEP-0000.md` document.
 | [cEP-0013](cEP-0013.md) | Cobot Enhancement and porting      | This cEP describes about the new features that are to be added to cobot as a part of the [GSoC project](https://summerofcode.withgoogle.com/projects/#4913450777051136).                                                                              |
 | [cEP-0014](cEP-0014.md) | Generate relevant coafile sections | This cEP proposes a framework for coala-quickstart to generate more relevant `.coafile` sections based on project files and user preferences.                                                                                                         |
 | [cEP-0019](cEP-0019.md) | Meta-review System                 | This cEP describes the details of the process of implementing the meta-review system as a part of the [GSoC'18 project](https://summerofcode.withgoogle.com/projects/#5188493739819008).                                                              |
+
```
Comment (sangamcse) [inline README.md]: No need to break line.

File: README.md [hunk 2]
```diff
@@ -19,3 +19,5 @@ in the `cEP-0000.md` document.
...
+| [cEP-0020](cEP-0020.md) | Newcomer Gamification system       | This cEP describes a gamification system to automate the recognition of activities performed by people who join the coala community, and encourage them to progress towards obtaining the role of "Developer" in the community. This cEP is a part of the [GSoC'18 project](https://summerofcode.withgoogle.com/projects/?sp-search=shrikrishna#5892040252981248).                                                              |
```
Comment (sangamcse) [inline README.md]: You mean https://summerofcode.withgoogle.com/projects/#5892040252981248 link?

**Review by sangamcse — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (sangamcse) [reply README.md]: Also, break the line to make 80 line length.

**Comment by sangamcse** [comment]
@gitmate-bot rebase

**Review by RaiVaibhav — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -5,21 +5,22 @@ in the `cEP-0000.md` document.
...
+| [cEP-0019](cEP-0019.md) | Meta-review System                 | This cEP describes the details of the process of implementing the meta-review system as a part of the [GSoC'18 project](https://summerofcode.withgoogle.com/projects/#5188493739819008).                                                                                                                                                     |
+| [cEP-0020](cEP-0020.md) | Newcomer Gamification system       | This cEP describes a gamification system to automate the recognition of activities performed by people who join the coala community, and encourage them to progress towards obtaining the role of "Developer" in the community. This cEP is a part of the [GSoC'18 project](https://summerofcode.withgoogle.com/projects/#5892040252981248). |
```
Comment (sangamcse) [inline README.md]: You have made 400 characters and changed whole table. Can you make your description short?

Comment (sks444) [reply README.md]: > Can you make your description short?

Yes, I can, I can make it something like:
```
This cEP describes the details of the process of implementing the newcomer gamification system as a part of the GSoC'18 project.
```
But I am not sure as that would take away the feel of the whole sentence :smile: 

Let @jayvdb, decide that, as he is also my mentor ;)

Comment (ksdme) [reply README.md]: Maybe consider something like:

This cEP describes a gamification system to help in recognition of activities and to encourage members who want to join our community obtain the role of "Developer". This cEP is a part of the [GSoC'18 project](https://summerofcode.withgoogle.com/projects/#5892040252981248).

Comment (jayvdb) [reply README.md]: agree, find a shorter description

**Review by jayvdb — CHANGES_REQUESTED** [review]

**Review by jayvdb — APPROVED** [review]

**Comment by jayvdb** [comment]
ack https://github.com/coala/cEPs/pull/166/commits/5c00c74ab68d8fc5a20e1d7df6a35fcd345d3b17

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**